### PR TITLE
added documentation and folder for twitter bot

### PR DESCRIPTION
**Pull request by edsun123**
I've added readme in each directory (inside flagging_site), that can be expanded on what each file does. (They can obviously read it inside the file but the readme makes it easier to get a general overview what each folder does) I've added twitter bot but I do not what should the output be so I have a basic update_status functionality (posts a tweet). I'm experimenting if users can tweet back to twitter to get different outputs (get cwa website, table of the data, etc.)

### CODE REVIEW

**Comment by dwreeves** [comment]
First of all, there's a lot of excellent stuff here. Great job documenting it!

I will get to the rest of your contributions soon, but I want to make a point about one particular thing. Sorry that it's so long, I just want to clarify my thought process.

----

I don't think we'll need a whole folder for a Twitter bot. It should fit into a single file called `twitter_bot.py`, or something like that. The basis of my argument for why is outlined below:

As far as I can tell, there are 5 good reasons why you'd want to use multiple files and multiple folders to build a Python project:

- **Organization:** i.e. to separate out chunks of code into conceptual units.
- **Isolation of independently operable components:** The `data` folder in particular has a lot of stuff in it that operates in isolation of everything else.
- **Isolation of unused or lazy-loaded components:** Certain parts of the code are not always loaded depending on the context. In particular, the `cyanobacteria.py` module is not loaded into production.
- **Git:** (For collaborative projects.) Under the git graph model, it's much easier to merge code when people work on separate files.
- **Convention:** (Not always applicable, but is in this case.) We're following a design pattern that is conventional of specific Flask apps.

Now, how does that apply in the case of a Twitter bot? Let's see. First, we need to define how the website should ultimately behave and the Twitter bot's role in that ecosystem:

- The website should update the model once an hour using a cron job in a task queue framework. (The framework I'm leaning toward is Huey, as it has a very simple API and builtin cron support, but I don't think the exact framework is important.)
- The cron job updates the database that everything else calls from.
- The Twitter bot should either (1) update every time the model does during daylight hours / when the Charles River is open to the public for boating or swimming; or (2) send out a tweet once a day as a cron job. We'll talk with CRWA about the specifics here.

The Twitter bot actually occupies a relatively small role in the ecosystem: it's just a way to send out notifications about data that we're already collecting and processing at other points in the ETL pipeline. In other words, it's just something that gets tacked onto the periphery of the ETL pipeline, for which we have plenty of other files. Note that whatever we do for a Twitter bot should use the same task queue framework we use for everything else; it doesn't make sense to use multiple task queue frameworks, that would be too bloated.

Now let's evaluate it in the context of the 5 points listed above:

- Organization: We're already organizing the task queue framework and database into their own respective files, and this is the bulk of the Twitter app.
- Isolation of independently operable components: The Twitter bot does not introduce independently operable components to the code, other than itself.
- Isolation of unused or lazy-loaded components: Not applicable here.
- Git: I don't suspect multiple people will need to work on this at the same time.
- Convention: Not applicable here.

For all these reasons, 1 file makes the most sense.

Do you agree or disagree with anything I said?

File: flagging_site/twitter/tweet_bot.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    auth.set_access_token(ACCESS_KEY, ACCESS_SECRET)
+    api = tweepy.API(auth)
+
+
+    #print message on terminal and update status 
+    # to twitter handle
+    print ('Tweeting' + msg) 
+    api.update_status(msg)
+
+    return msg
```
Comment (dwreeves) [inline flagging_site/twitter/tweet_bot.py:36]: This file looks good to me!

File: flagging_site/static/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# content of static
+
+- images : contains all iamges
+- style.css : style sheet
```
Comment (dwreeves) [inline flagging_site/static/README.md:4]: I wouldn't include a README in `static`. As long as you know what the file is (i.e. a file for static resources that Flask treats as special), it's fairly self-explanatory everything that's in it. Also all files in `static` are accessible when the site runs, as far as I am aware.

File: flagging_site/data/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Content of data
+
+- _store : contains pickle files used to test webpage offline using data provided already from usgus.pickle and hobolink.pickle
+- __init__.py : required to treat directory as a package
+- database.py : file handling database connection
+- hobolink.py : retrieve hobolink by requesting a response and parsing data from hobolink 
+- keys.py : handles access and tokens for hoblink and usgs API. Vault.zip provides keys.yml that provides the credentials
+- model.py : outputs table model by processing usgs and hobolink data 
+- task_queue : set up task queue 
+- usgs.py : retrieve hobolink by requesting a response and parsing data from usgs
```
Comment (dwreeves) [inline flagging_site/data/README.md:10]: Very good! 🔥

File: flagging_site/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# flagging_site Content 
+
+  - blueprints: contains files that are modular applications added to main flask application
+  - data: contains files that retrieves and cleans the hobolink and usgs data
+  - static: contains images and css/style sheet
+  - templates: contains the html pages that define the web page structure
+  - __init__.py: required to treat directory as a package
+  - app.py: main flask application
+  - config.py: file holding the config options for flask and 
+  - vault.zip: contains credentials necessary to accessing hoblink and usgs api
```
Comment (dwreeves) [inline flagging_site/README.md:10]: Very good! And you definitely understand the app very well, which is no small feat as there's a lot going on. 🎉 On line 9 it looks like you forgot to finish the sentence.

**Comment by edsun123** [comment]
fixed issue: Convert Yaml to Json

**Comment by edsun123** [comment]
Checking if i works as a website offline

**Comment by dwreeves** [comment]
Forgot to say: LGTM!

**Pull request merged**